packages/component-library/src/Input.tsx

```diff
@@ -56,6 +57,7 @@ export function Input({
   onChangeValue,
   onUpdate,
   className,
+  useKeyDown = false,
```

Comment:
Doing changes to the base `Input` component has a very large blast radius. This addition poses significant risk. As such: it cannot be taken lightly.

Conceptually - I'm not a fan of this boolean flag that amends the existing logic. Why is this necessary? Are there other ways to achieve something similar without creating conditional logic for key up/down handling?